[CI] CoreWithMultipleProjectsClientYamlTestSuiteIT test {yaml=data_stream/150_tsdb/created the data stream} failing

**Build Scans:**
- [elasticsearch-intake #19862 / part2](https://gradle-enterprise.elastic.co/s/qug2taa5x5zau)
- [elasticsearch-pull-request #61777 / part-2](https://gradle-enterprise.elastic.co/s/zb66hfazmdpyo)
- [elasticsearch-pull-request #61735 / part-2](https://gradle-enterprise.elastic.co/s/6raxjl7kw77rg)
- [elasticsearch-pull-request #61578 / part-2](https://gradle-enterprise.elastic.co/s/ws3dceafrmwom)
- [elasticsearch-pull-request #61224 / part-2](https://gradle-enterprise.elastic.co/s/d7v3tepb6kwae)

**Reproduction Line:**
```
./gradlew ":x-pack:qa:multi-project:core-rest-tests-with-multiple-projects:yamlRestTest" --tests "org.elasticsearch.multiproject.test.CoreWithMultipleProjectsClientYamlTestSuiteIT.test {yaml=data_stream/150_tsdb/created the data stream}" -Dtests.seed=3667756CA22BBFD3 -Dtests.locale=sq-AL -Dtests.timezone=Africa/Bangui -Druntime.java=23
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.multiproject.test.CoreWithMultipleProjectsClientYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(test%20%7Byaml%3Ddata_stream%2F150_tsdb%2Fcreated%20the%20data%20stream%7D),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [data_stream/150_tsdb:111]: 
Expected: "2021-04-29T00:00:00.000Z"
     but: was "2025-03-11T16:08:51.000Z"
```

**Issue Reasons:**
- [main] 5 failures in test test {yaml=data_stream/150_tsdb/created the data stream} (0.8% fail rate in 617 executions)
- [main] 4 failures in step part-2 (2.1% fail rate in 192 executions)
- [main] 4 failures in pipeline elasticsearch-pull-request (2.1% fail rate in 192 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/modules/data-streams/src/yamlRestTest/resources/rest-api-spec/test/data_stream/150_tsdb.yml
+++ b/modules/data-streams/src/yamlRestTest/resources/rest-api-spec/test/data_stream/150_tsdb.yml
@@ -108,7 +108,8 @@ created the data stream:
   - match: { data_streams.0.hidden: false }
   - match: { data_streams.0.system: false }
   - match: { data_streams.0.time_series.temporal_ranges.0.start: 2021-04-28T00:00:00.000Z }
-  - match: { data_streams.0.time_series.temporal_ranges.0.end: 2021-04-29T00:00:00.000Z }
+  # We can't match on time_series.temporal_ranges.0.end in this test because UpdateTimeSeriesRangeService updates it every 5 minutes, so every
+  # once in a while it is not what we expect it to be. See #124575.
   - set:  { data_streams.0.indices.0.index_name: backing_index }
 
   - do:

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -212,9 +212,6 @@ tests:
 - class: org.elasticsearch.multiproject.test.CoreWithMultipleProjectsClientYamlTestSuiteIT
   method: test {yaml=data_stream/190_failure_store_redirection/Redirect ingest failure in data stream to failure store}
   issue: https://github.com/elastic/elasticsearch/issues/124518
-- class: org.elasticsearch.multiproject.test.CoreWithMultipleProjectsClientYamlTestSuiteIT
-  method: test {yaml=data_stream/150_tsdb/created the data stream}
-  issue: https://github.com/elastic/elasticsearch/issues/124575
 - class: org.elasticsearch.xpack.restart.MLModelDeploymentFullClusterRestartIT
   method: testDeploymentSurvivesRestart {cluster=OLD}
   issue: https://github.com/elastic/elasticsearch/issues/124160

